plugins/systemd/systemd_status

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3 -tt
```

Comment:
Why `-tt`?

Also please use `/usr/bin/env python` in order to allow virtualenv usage.